avaframe/ana4Stats/probAna.py

```diff
@@ -148,55 +165,63 @@ def probAnalysis(avaDir, cfg, module, parametersDict='', inputDir=''):
 
     # fetch all result files and filter simulations according to parametersDict
     simNameList = cfgUtils.filterSims(avaDir, parametersDict, specDir=inputDir)
-    if inputDir == '':
-        inputDir = avaDir / 'Outputs' / modName / 'peakFiles'
-        flagStandard = True
-        peakFilesDF = fU.makeSimDF(inputDir, avaDir=avaDir)
-    else:
-        inputDirPF = inputDir / 'peakFiles'
-        peakFilesDF = fU.makeSimDF(inputDirPF, avaDir=avaDir)
-
-    # get header info from peak files - this should be the same for all peakFiles
-    header = IOf.readASCheader(peakFilesDF['files'][0])
-    cellSize = header['cellsize']
-    nRows = header['nrows']
-    nCols = header['ncols']
-    xllcenter = header['xllcenter']
-    yllcenter = header['yllcenter']
-    noDataValue = header['noDataValue']
-
-    # Initialise array for computations
-    probSum = np.zeros((nRows, nCols))
-    count = 0
-
-    # Loop through peakFiles and compute probability
-    for m in range(len(peakFilesDF['names'])):
-
-        # only take simulations that match filter criteria from parametersDict
-        if peakFilesDF['simName'][m] in simNameList:
-            # Load peak field for desired peak field parameter
-            if peakFilesDF['resType'][m] == cfg['GENERAL']['peakVar']:
-
-                # Load data
-                fileName = peakFilesDF['files'][m]
-                data = np.loadtxt(fileName, skiprows=6)
-                dataLim = np.zeros((nRows, nCols))
-
-                log.info('File Name: %s , simulation parameter %s ' % (fileName, cfg['GENERAL']['peakVar']))
-
-                # Check if peak values exceed desired threshold
-                dataLim[data > float(cfg['GENERAL']['peakLim'])] = 1.0
-                probSum = probSum + dataLim
-                count = count + 1
-
-    # Create probability map ranging from 0-1
-    probMap = probSum / count
-    unit = pU.cfgPlotUtils['unit%s' % cfg['GENERAL']['peakVar']]
-    log.info('probability analysis performed for peak parameter: %s and a peak value threshold of: %s %s' % (cfg['GENERAL']['peakVar'], cfg['GENERAL']['peakLim'], unit))
-    log.info('%s peak fields added to analysis' % count)
-
-    # # Save to .asc file
-    avaName = avaDir.name
-    outFileName = '%s_probMap%s.asc' % (avaName, cfg['GENERAL']['peakLim'])
-    outFile = outDir / outFileName
-    IOf.writeResultToAsc(header, probMap, outFile)
+
+    # initialize flag if analysis has been performed or e.g. no matching files found
+    analysisPerformed = False
+    if simNameList != []:
```

Comment:
The is no path in here if simListName is empty, and the rest is indented just because of this. 
So I suggest adding a second return here with a simple 
if simNamelinst is empty
  analysisPerformed = False
  return ana....
And then don't indent the rest....

Reply:
done